custom-connectors/M365Lighthouse/Readme.md

```diff
@@ -0,0 +1,32 @@
+# Microsoft 365 Lighthouse Connector
```

Comment:
Kindly update readme.md file as per the below template. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md